lib/utilities/valid-platform-version.js

```diff
@@ -8,5 +8,6 @@ module.exports = function(version) {
          semver.satisfies(version, '^2.0.0')  || // io.js 2.x.y
          semver.satisfies(version, '^3.0.0')  || // io.js (dev)
          semver.satisfies(version, '^0.12.0') || // node 0.12.x
-         semver.satisfies(version, '^0.13.0');   // node 0.13.x
+         semver.satisfies(version, '^0.13.0') || // node 0.13.x
+         semver.satisfies(version, '^4.0.0');    // node 4.x.x
```

Comment:
@stefanpenner you right, this would be enough:

```
module.exports = function(version) {
  return semver.satisfies(version, '>=0.12.0');
};
```

I update and push it up.